### DISCUSSION TITLE
[Script request] Mailcow

### DISCUSSION THREAD

**Discussion by Felitendo** [community-scripts/ProxmoxVE#424] [Request script] [answered]
### Application Name

Mailcow

### Website

https://github.com/mailcow/mailcow-dockerized

### Description

Mailcow is a selfhosted E-Mail Server that makes hosting E-Mail Servers as easy as it can be. More information here: https://docs.mailcow.email/

### Due Diligence

- [X] I have searched existing [scripts](https://community-scripts.github.io/Proxmox/scripts) and found no duplicates.
- [X] I have searched existing [discussions](https://github.com/community-scripts/ProxmoxVE/discussions?discussions_q=) and found no duplicate requests.
- [X] This is not a game-related request.

**Comment by Dvalin21** [comment]
+1

**Comment by FinFox1** [comment]
Found an installation guide (german) for LXC using Docker inside the LXC. Youtube Video: https://www.youtube.com/watch?v=Zo2Hsh_oLgk

Github Repo with instruction: https://github.com/RPiList/Videolinks/blob/main/Videos/1eur-LXC-Mailcow

Maybe this one helps.

**Reply by MickLesk** [reply]
not really, its just an docker in an lxc

**Comment by blitzu** [comment]
+1

**Comment by JesseHoekema** [comment]
+1

**Comment by michelroegl-brunner** [comment, marked as answer]
That is better suited as a Docker, as it has so much dependencies and moving parts. This would be a nightmare to maintain.
